### ISSUE TITLE
Uncaught TypeError: Object(...) is not a function in ionic-admob/dist/index.js (vendor.js:78042)

### ISSUE THREAD

**Issue by DooDiddle** [admob-plus/admob-plus#33]
Uncaught TypeError: Object(...) is not a function
    at vendor.js:78023
    at Module../node_modules/ionic-admob/dist/index.js (vendor.js:78042)
    at __webpack_require__ (runtime.js:84)
    at Module../src/app/app.module.ts (main.js:1701)
    at __webpack_require__ (runtime.js:84)
    at Module../src/main.ts (main.js:1780)
    at __webpack_require__ (runtime.js:84)
    at Object.0 (main.js:1802)
    at __webpack_require__ (runtime.js:84)

I followed all instructions from the guide. Started with a ionic 4 tabs app. Appears to be an error coming from admob, can you please help?

**Comment by DooDiddle** [comment]
npm WARN ionic-admob@0.7.0 requires a peer of @ionic-native/core@^4.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN ionic-admob@0.7.0 requires a peer of rxjs@^5.0.1 but none is installed. You must install peer dependencies yourself.

My Package.json in ionic 4 is:
    "@ionic-native/core": "5.0.0-beta.21",
    "rxjs": "^6.2.2",
    "rxjs-compat": "^6.3.3",

**Comment by ratson** [comment]
@DooDiddle You may try out the "not documented" `@admob-plus/ionic` package, it is designed to work with Ionic v4.

**Comment by DooDiddle** [comment]
Thank you, I still have an issue - I can now see test ad, but not the live add. The banner rectangle doesn't show up for non-test banner ad.
I removed admob-ionic completely, installed @admob-plus/ionic, and replaced imports as:
import { AdMob } from "@admob-plus/ionic"

Then inside package.json I got:
      "cordova-admob-plus": {
        "APP_ID_ANDROID": "ca-app-pub-123/123",
        "PLAY_SERVICES_VERSION": "16.0.0"
      }

then inside the app in the constructor:
    platform.ready().then(() => {
        admob.banner.show({ id: "ca-app-pub-123123" }));
}

I don't get errors, it just doesn't appear at the bottom at all. Is there something else I need?
